### PR TITLE
Add hibernate

### PR DESCRIPTION
**Pull request by Vladlis**

### CODE REVIEW

File: web-project/src/main/java/com/epam/webproject/dao/impl/AuthorDaoImpl.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    @Override
+    public Author getById(int id) {
+        return null;
+    }
+
+    @Override
+    public Author save(Author author) {
+        entityManager.persist(author);
+        entityManager.flush();
```
Comment (sqshq) [inline web-project/src/main/java/com/epam/webproject/dao/impl/AuthorDaoImpl.java]: why do we need `flush` here?

File: web-project/src/main/java/com/epam/webproject/dao/impl/AuthorDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    @PersistenceContext
+    private EntityManager entityManager;
+
+    @Override
+    public Author getById(int id) {
+        return null;
+    }
+
+    @Override
+    public Author save(Author author) {
```
Comment (sqshq) [inline web-project/src/main/java/com/epam/webproject/dao/impl/AuthorDaoImpl.java]: where in the project do you invoke this method?

Comment (Vladlis) [reply web-project/src/main/java/com/epam/webproject/dao/impl/AuthorDaoImpl.java]: This method is not used due to the poor ui. All unused methods are removed from the repository layer.

File: web-project/src/main/java/com/epam/webproject/dao/impl/AuthorDaoImpl.java (hunk 1 above)
Comment (Vladlis) [reply web-project/src/main/java/com/epam/webproject/dao/impl/AuthorDaoImpl.java]: Not needed, as auto commit is set by default

**Pull request merged**